virttest/utils_kernel_module.py

```diff
@@ -48,12 +81,38 @@ def __init__(self, module_name):
         """Create kernel module handler"""
 
         self._module_name = module_name
+        self._module_path = os.path.join('/sys/module/',
+                                         self._module_name.replace('-', '_'))
+        self._module_params_path = os.path.join(self._module_path, 'parameters')
         self._was_loaded = None
-        self._loaded_config = None
```

Comment:
I removed it because we have backup the config, check it dynamically is more reasonable, if they do not match, restore it. Otherwise, when reloading `kvm`, the kvm probe module failed to reload.
But since we are talking about the current design, I will take care of it.